lib/web_ui/lib/src/engine/html_image_codec.dart

```diff
@@ -33,15 +33,15 @@ class HtmlCodec implements ui.Codec {
     // we add 0/100 , 100/100 progress callbacks to enable loading progress
     // builders to create UI.
     if (chunkCallback != null) {
-      chunkCallback(0, 100);
+      chunkCallback!(0, 100);
```

Comment:
It is inside chunkCallback!=null , maybe its a temporary limitation that you need chunkCallback! ?